random_student_marks_mysql.php

```diff
@@ -17,37 +16,40 @@
     $teachingRandomSubjects[$i] = $teachingSubjects[$teachingRandomSubject];
 }
 $uniqueStudentNames = array();
-foreach ($studentNames as $key => $value) {
-    if (!in_array($value, $uniqueStudentNames)) {
-        $uniqueStudentNames[] = $value;
+foreach ($studentNames as $names) {
+    if (!in_array($names, $uniqueStudentNames)) {
+        $uniqueStudentNames[] = $names;
     }
 }
 $uniqueTeachingRandomSubjects = array();
-foreach ($teachingRandomSubjects as $key => $value) {
-    if (!in_array($value, $uniqueTeachingRandomSubjects)) {
-        $uniqueTeachingRandomSubjects[] = $value;
+foreach ($teachingRandomSubjects as $key => $subject) {
+    if (!in_array($subject, $uniqueTeachingRandomSubjects)) {
+        $uniqueTeachingRandomSubjects[] = $subject;
     }
 }
-foreach ($uniqueStudentNames as $key => $value) {
+foreach ($uniqueStudentNames as $key => $names) {
     $uniqueStudentName = explode(",", $uniqueStudentNames[$key]);
     $sqlQuerry = "INSERT INTO students (name, surname) VALUES ('$uniqueStudentName[0]', '$uniqueStudentName[1]')";
     mysqli_query($connection, $sqlQuerry);
 }
-foreach ($uniqueTeachingRandomSubjects as $key => $value) {
+foreach ($uniqueTeachingRandomSubjects as $key => $subject) {
     $sqlQuerry = "INSERT INTO teaching_subjects (teachingSubject) VALUES ('$uniqueTeachingRandomSubjects[$key]')";
     mysqli_query($connection, $sqlQuerry);
 }
 $counterStudent = count($uniqueStudentNames);
 $counterSubject = count($uniqueTeachingRandomSubjects);
+define("TIME_STAMP_YEAR_2000", "946684800");
```

Comment:
Dažniausiai konstantos aprašomos failo arba klasės (objektiniame programavime) viršuje. Tikriausiai dėl to, kad konstantos viso failo vykdymo metu savo reikšmės nepakeis + patogu kai visos jos aprašytos vienoje vietoje + taip aprašant sudėtinga padaryti kad pvz define būtų kviečiamas ciklo viduje - kas nėra labai logiška/optimalu